easybuild/main.py

```diff
@@ -213,149 +213,20 @@ def clean_exit(logfile, tmpdir, testing, silent=False):
     sys.exit(0)
 
 
-def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
+def rest_of_main(orig_paths, options, cfg_settings, modtool, testing, init_session_state, hooks, do_build):
```

Comment:
I would rename `rest_of_main` to `process_eb_args`, rename `orig_paths` to `args` , and also tweak the description of this function:

```python
def process_eb_args(args, options, ...):
    """
    Remainder of main function, actually process provided arguments (list of files/paths),
    according to specified options.

    :param args: list of arguments that were specified to 'eb' command (or an easystack file);
                 includes filenames/paths of files to process
                 (mostly easyconfig files, but can also includes patch files, etc.)

    ...
    """
```

Reply:
fixed in e107a4b55670c9386ed518e91cc39dc431fa6267